lib/stitches/render_timestamps_in_iso8601_in_json.rb

```diff
@@ -3,7 +3,7 @@
 class ActiveSupport::TimeWithZone
   # We want dates to be a) in UTC and b) in ISO8601 always
   def as_json(options = {})
-    utc.iso8601
+    utc.iso8601(ActiveSupport.time_precision)
```

Comment:
An alternate implementatin could be

```ruby
def as_json(options = {})
  if utc?
    super
  else
    utc.as_json
  end
end
```

Reply:
Here's the latest Rails implementation of `TimeWithZone#as_json` for reference:

https://github.com/rails/rails/blob/a82df4c5e0e26deed76ddc37ffd95a86455d8265/activesupport/lib/active_support/time_with_zone.rb#L167-L173

Reply:
What I'm wondering about is, what was this little patch meant to accomplish? Forcing the timezone or the serialization format?

Reply:
Also, is there any reason now that we can't just rely on Rails' behavior?